README.adoc

```diff
@@ -3,9 +3,11 @@ image:https://img.shields.io/badge/iOS-Compatible-green.svg[]
 image:https://img.shields.io/badge/Ionic%20Version-3-green.svg[]
 image:https://img.shields.io/badge/Angular%20Version-4-green.svg[]
 image:https://badges.frapsoft.com/typescript/code/typescript.svg?v=101[]
+image:https://circleci.com/gh/aerogear/cordova-showcase-template.svg?&style=shield&circle-token=6977be9bf600b6b8770d784be11913497629dc8b[]
```

Comment:
is it required/okay that this token's here in public?

Reply:
nvm saw @aliok's comment

Reply:
@paolobueno good point. It's not!

Reply:
It works without that as well.

Reply:
:+1: I tried the url without the token and it seemed to work too, so best not to have it

Reply:
nope. it works for **logged in** users. 

https://circleci.com/docs/1.0/status-badges/
> (You can also link to the image without providing a circle-token, in which case the image will only appear for people who are currently logged-in to circleci.com!)